### PR TITLE
Local cache TTL factor 

### PR DESCRIPTION
**Pull request by rem1niscence**
Enhance the local ttl by adding a custom factor based in the remote ttl given

### CODE REVIEW

File: src/services/cache.ts [hunk 1]
```diff
@@ -55,7 +62,7 @@ export class Cache {
   }
 
   async expire(key: string, ttlSeconds: number) {
-    await this.local.expire(key, ttlSeconds)
+    await this.local.expire(key, this.getLocalTTlL(ttlSeconds))
```
Comment (nymd) [inline src/services/cache.ts]: there's an extra 1 in this function name

**Review by nymd — CHANGES_REQUESTED** [review]

File: src/services/cache.ts (hunk 1 above)
Comment (rem1niscence) [reply src/services/cache.ts]: l you mean haha but you're right, on it

**Review by nymd — APPROVED** [review]
did you set the prod secret

**Comment by rem1niscence** [comment]
> did you set the prod secret

yes

**Pull request merged**